packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -349,16 +363,21 @@ function BackgroundImageControls( {
 				backgroundSize: sizeValue,
 			} )
 		);
+		setIsUploading( false );
 	};
 
+	// Drag and drop callback, restricting image to one.
 	const onFilesDrop = ( filesList ) => {
+		if ( filesList?.length > 1 ) {
+			onUploadError(
+				__( 'Only one image can be used as a background image.' )
+			);
+			return;
+		}
 		mediaUpload( {
 			allowedTypes: [ IMAGE_BACKGROUND_TYPE ],
 			filesList,
 			onFileChange( [ image ] ) {
-				if ( isBlobURL( image?.url ) ) {
-					return;
-				}
```

Comment:
Just double checking — this can be removed because it's already being handled early on in `onSelectMedia`, yeah?

Reply:
Sorry, I meant to write a note about that. Yeah, the same check is on `onSelectMedia`. 👍🏻